### DISCUSSION TITLE
cannot use affine cloud

### DISCUSSION THREAD

**Discussion by xieding001** [toeverything/AFFiNE#7452] [Q&A] [answered]
After self-host affine, I run localhost:3010. The interface appears, but there's missing "login with email " button.
what is the problem?
<img width="644" alt="983fc23862c6aff6116d6cc0ad8cc84" src="https://github.com/toeverything/AFFiNE/assets/8392251/f46bd1ab-b2ae-4d46-bac7-049bb0c48246">
there is no error while deploying.
<img width="1920" alt="20240709110839" src="https://github.com/toeverything/AFFiNE/assets/8392251/ba33dd34-4a04-4fc8-ac57-52ee7449cf0d">

**Comment by forehalo** [comment, marked as answer]
this might caused by the unstable connection to load cloudflare captcha.

**Reply by xieding001** [reply]
thanks, you are right. My VM cannot load https://challenges.cloudflare.com/turnstile/v0/b/191f93ebdf8e/api.js, but my host machine can load it. Do you know why?

**Reply by forehalo** [reply]
probably the proxy config isn't working in your VM

**Reply by xieding001** [reply]
yes, probably the DNS is the problem.
I met a new problem, that I set the admin username and password but when logging it shows 'invalid password'
